sections/introduction.include

```diff
@@ -825,11 +825,11 @@
   :: While it is possible to use presentational markup in a way that provides users of assistive
       technologies (ATs) with an acceptable experience (e.g., using ARIA), doing so is significantly
       more difficult than doing so when using semantically-appropriate markup. Furthermore, even
-      using such techniques doesn't help make pages accessible for non-AT non-graphical users, such
-      as users of text-mode browsers.
+      using such techniques doesn't help make pages accessible for users of non-AT, non-graphical
+      comsumer/users, such as text-mode browsers.
```

Comment:
How about: 

Furthermore, presentational markup does not provide accessibility for users of non-AT, non-graphical user agents (such as text-mode browsers).